public/apps/configuration/utils/display-utils.tsx

```diff
@@ -51,3 +55,48 @@ export function renderCustomization(reserved: boolean) {
     </EuiText>
   );
 }
+
+export function renderSidePanel() {
+  return (
+    <EuiPageSideBar>
+      <NavPanel items={ROUTE_LIST} />
+    </EuiPageSideBar>
+  );
+}
+
+export function renderBreadcrumbs(resourceType: ResourceType, pageTitle?: string) {
+  if (!pageTitle) {
+    return (
+      <EuiBreadcrumbs
+        breadcrumbs={[
+          {
+            text: 'Security',
+            href: buildHashUrl(),
+          },
+          {
+            text: ROUTE_MAP[resourceType].name,
+            href: buildHashUrl(resourceType),
+          },
+        ]}
+      />
+    );
+  }
+
+  return (
+    <EuiBreadcrumbs
+      breadcrumbs={[
+        {
+          text: 'Security',
+          href: buildHashUrl(),
+        },
+        {
+          text: ROUTE_MAP[resourceType].name,
+          href: buildHashUrl(resourceType),
+        },
+        {
```

Comment:
These 2 routes looks quite same. Suggest we define the same part as a base value, and concat the third one if title provided

Reply:
True. Switched to concat